demo/bulk_insert/bulk_insert.py

```diff
@@ -118,9 +135,18 @@ def ProcessNodes(nodes_csv_files):
 			# The first row of a label CSV contains its attributes
 			descriptor.attributes = header_row
 
+
 			# Update counts
 			header_len = len(header_row)
 			descriptor.attribute_count += header_len
+
+			# Count number of entities (line count - 1 for header row)
+			descriptor.count_entities(csvfile)
```

Comment:
If possible have descriptor constructor (__init__) call count_entities if we can pass csvfile as a parameter.